genai-perf/tests/test_perf_analyzer_config.py

```diff
@@ -110,8 +110,6 @@ def test_command_creation(self):
             "999",
             "--request-count",
             "128",
-            "--warmup-request-count",
-            "0",
```

Comment:
GAP no longer writes `--warmup-request-count` when its value is 0. It's unnecessary from PA perspective.

https://github.com/triton-inference-server/perf_analyzer/pull/366/files#diff-d1c5f6fe3374963d887eec07bbc9270be4be7ca81e1c6ccf2632c50027204eb3R249-R253